### DISCUSSION TITLE
Opschalingsanalyse vraag 3: Is de gemeente voldoende toegerust voor een doeltreffende uitvoering?

### DISCUSSION THREAD

**Discussion by marcvanandel** [kadaster-labs/sensrnet-home#253] [Pilot Gemeente]
Opschalingsanalyse vraag 3: **Is de gemeente voldoende toegerust voor een doeltreffende uitvoering?**

**Comment by WOostdam** [comment]
Om te kunnen werken met het MVP? Nee.
Om een registratie op te zetten? Ja

**Reply by rsleebos** [reply]
Wat ontbreekt er aan het MVP om hier wél mee te kunnen werken?

**Comment by WOostdam** [comment]
Wat bij ons ontbreekt, is dat wij als gemeente op dit moment niet beschikken over een platform (hardware + software, die haven-compatible is) waarop wij de MVP kunnen “deployen”, kunnen installeren en activeren. Het gaat dus niet zozeer om de functionaliteit van MVP die ons parten zou spelen.

Ok, meedoen met Tilburg is een optie. Installeren op de omgeving waarop we Signalen draaien is ook een optie. Maa, in onze ogen, niet ideaal. Liever zelf.

Meedoen met anderen kost ook gewoon geld. Zelf doen ook. Dan liever zelf doen? Maar zelf doen vraagt ook kennis en capaciteit om zo’n platform in te richten en te onderhouden. Over deze vragen/afwegingen hebben we op dit moment nog niet het antwoord. Antwoord dus in termen van geld, kennis en capaciteit.
Maar we denken er wel erg over na. Of het antwoord op tijd komt om nog een rol te spelen bij het testen van de MVP weet ik op dit moment niet.

Groet,
Walter Oostdam

**Reply by marcvanandel** [reply]
Naast de techniek valt er nog veel meer te benoemen:
- techniek niet alleen, maar ook
- organisatie
- financiering

**Comment by mikegroels** [comment]
De vraag is dus ook:
- Wat is doeltreffend? (het behoorlijk kunnen registreren en beheren van de sensors? of voldoen aan de verwachtingen m.b.t. transparantie en tellen de kosten dan ook nog mee?)
- Wat is uitvoering? (puur asset management, of van installatie tot transparantie en alles wat daartussen ligt)

**Comment by marcvanandel** [comment]
Techniek is wel een belemmering ... of op z'n minst onduidelijk. Maar naast de techniek zijn er nog vele andere onduidelijkheden:

De plek en relevantie die sensoren hebben, is onduidelijk. De verantwoordelijkheden die we als gemeente hebben is onduidelijk (is het - deels - verplicht vanuit AVG tbv transparantie)? Is het alleen vanuit asset management relevant?

Er is te weinig aandacht in het algemeen voor sensoren, het beheer daarvan in 'asset management' zin en ook in verantwoordelijkheden. En wat betekent dit dan voor de gemeente? En voor afdelingen?

**Reply by marcvanandel** [reply]
Als asset management de primaire driver is, dan is de integratie met asset management systemen een hoge prioriteit. Dat zou kunnen betekenen dat de nationale registratie minder prio heeft dan die integratie. Het vraagt een andere driver om nationale registratie tbv transparantie te realiseren.

**Comment by marcvanandel** [comment]
Als je een digitale stad wil zijn, dan _moet_ je wel iets met sensoren en het registreren van sensoren gaan doen. Zo zal dat in de grotere gemeenten en zeker digitaal gerichte gemeenten wel als behoefte naar boven komen (vroeg of laat).

Tegelijk zijn er processen, procesbegeleiding, tools, ondersteuning en functionaliteiten noodzakelijk om te ondersteunen dat sensoren gemakkelijk geregistreerd kunnen worden.

Een stroomschema hoe je als gemeente komt tot registratie zou hierbij kunnen helpen. (niet alleen tools en techniek, maar ook organisatie en beleid)

**Reply by marcvanandel** [reply]
Er zou een platform moeten zijn zodat je [als gemeente]  gemakkelijk aan je verantwoordelijkheid zou kunnen voldoen.

(en hoeveel werk, geld en aandacht mag dat kosten?)
